azerothcore/azerothcore-wotlk
# fix(DB/events): Fishing Extravaganza wrong start time

## Changes Proposed:
-  Fishing Extravaganza should start at 14:00 server time, not 15:00
-  Push back all the STV fishing event start times by 1 hour.

## Issues Addressed:

- Closes azerothcore#5450

## SOURCE:


## Tests Performed:

- Set my local server time to just before 14:00 on a Sunday.
- Went to Booty Bay and waited until 14:00. At 14:01-14:02 the NPC announced the fishing tournament had begun, and the turn-in NPC spawned (id: 15077)
- Flew around the coasts of STV and observed that tastyfish pools were now spawning

## How to Test the Changes:


1. Run ```SELECT * FROM `game_event` WHERE `eventEntry` IN (14, 15, 62);``` and ensure the start time for announce is at midnight, and the start times for fishing pools and turn-ins is at 14:00
2. Set server time to just before 14:00 on a Sunday and Wait in Booty Bay for the event to start.
3. At 14:00 tastyfish fishing pools should begin spawning in Stranglethorn Vale. At around 14:01 a zone-wide NPC announcement should occur stating that the tournament has begun.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
